docs/about/introduction.mdx

```diff
@@ -12,7 +12,8 @@ Meshtastic® is a project that enables you to use inexpensive LoRa radios as a l
 ### Features
 
 - Long range ([_166km record by PuzzledPancake_](/docs/overview/range-tests#current-record))
-- No phone required for mesh communication
+- No phone signal required for mesh communication
```

Comment:
I'd Rever this, as the functionality of the mess does not depend on a phone at all

Reply:
accepted